src/main/java/com/example/kinoxpback/repository/ShowingRepository.java

```diff
@@ -2,7 +2,14 @@
 
 import com.example.kinoxpback.model.Showing;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Date;
+import java.util.List;
 
 public interface ShowingRepository extends JpaRepository<Showing,Integer> {
+    @Query("SELECT s FROM Showing s WHERE s.showingDate = :date")
```

Comment:
s.showingDate needs to be s.showing_date

Reply:
Fixed